setup.py

```diff
@@ -345,6 +327,7 @@ def run(self):
     )
     + extras["retrieval"]
     + extras["modelcreation"]
+    + extras["tiktoken"]
```

Comment:
needed to test `py.test tests/models/llama/test_tokenization_llama.py::TikTokenIntegrationTests`, so it makes sense to be in the `testing` extra